packages/react/runtime/src/lynx/performance.ts

```diff
@@ -93,7 +93,11 @@ function beginPipeline(needTimestamps: boolean, pipelineOrigin: PipelineOrigin,
         break;
     }
 
-    lynx.performance?._onPipelineStart?.(globalPipelineOptions.pipelineID, globalPipelineOptions);
+    if (SystemInfo.engineVersion >= '3.0') {
```

Comment:
`SystemInfo.engineVersion` is `undefined` before 3.2. Use `SystemInfo.lynxSdkVersion` instead.
```suggestion
    if (SystemInfo.lynxSdkVersion >= '3.0') {
```

Reply:
changed to isSdkVersionGt(3, 1).